lib/SILOptimizer/PassManager/PassPipeline.cpp

```diff
@@ -101,7 +101,7 @@ SILPassPipelinePlan::getDiagnosticPassPipeline(SILOptions Options) {
   // disabled.
   if (Options.DebugSerialization) {
     addMandatoryDebugSerialization(P);
-    return P;
+    return const_cast<const SILPassPipelinePlan&>(P);
```

Comment:
I was just experimenting with ideas to fix the error. I can remove the delete decleration if you'd like